src/reducers/PatientReducer.js

```diff
@@ -76,6 +76,10 @@ export const PatientReducer = (state = patientInitialState(), action) => {
       return { ...state, currentPatient: null, creatingADR: false };
     }
 
+    case PATIENT_ACTIONS.REFRESH: {
+      return { ...patientInitialState() };
+    }
+
```

Comment:
think dispatching this action from the `batch` inside `patientUpdate` would be OK? not biggy, but nice to have the single state update

Reply:
Yes, thanks. that's gooder. Is this what you mean?
9ea8703d